hermeto/core/package_managers/cargo/main.py

```diff
@@ -221,8 +221,8 @@ def _run_cmd_watching_out_for_lock_mismatch(
             raise
 
 
-def _resolve_cargo_package(package_dir: RootedPath) -> chain[Component]:
-    """Resolve a single cargo package."""
+def _generate_sbom_components(package_dir: RootedPath) -> chain[Component]:
```

Comment:
Much better, yes, this is (overall) more consistent with the rest of the backends.